<!doctype html>
<html lang="en">

<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="author" content="">
  <meta name="description" content="CADD - Combined Annotation Dependent Depletion">
  <title>CADD - Combined Annotation Dependent Depletion</title>
  <link rel="icon" href="/static/icon.png">
  <link rel="stylesheet" href="/static/milligram.css">
</head>

<body>

  <main class="wrapper">

    <nav class="navigation">
      <section class="container">
        <a class="navigation-title" href="/">
          <img src="/static/icon.png" width=40 height=40 align=top>
          <h1 class="title">CADD - Combined Annotation Dependent Depletion</h1>
        </a>
        <ul class="navigation-list float-right">
          <li class="navigation-item">
            <a class="navigation-link" href="/news" title="Recent updates">News</a>
          </li>
          <li class="navigation-item">
            <a class="navigation-link" href="/score" title="Upload a vcf file for variant scoring">Score</a>
          </li>
          <li class="navigation-item">
            <a class="navigation-link" href="#popover-snv" data-popover>SNV</a>
            <div class="popover" id="popover-snv">
              <ul class="popover-list">
                <li class="popover-item"><a class="popover-link" href="/snv" title="Lookup for single SNV">Single</a>
                </li>
                <li class="popover-item"><a class="popover-link" href="/snv-range" title="SNV range lookup">Range</a>
                </li>
              </ul>
            </div>
          </li>
          <li class="navigation-item">
            <a class="navigation-link" href="/download" title="Data downloads for offline scoring">Downloads</a>
          </li>
          <li class="navigation-item">
            <a class="navigation-link" href="#popover-about" data-popover>About</a>
            <div class="popover" id="popover-about">
              <ul class="popover-list">
                <li class="popover-item"><a class="popover-link" href="/info" title="About CADD">Information</a></li>
                <li class="popover-item"><a class="popover-link" href="/api"
                    title="Information about the CADD API">API</a></li>
                <li class="popover-item"><a class="popover-link" href="/genome-browser"
                    title="Information about displaying CADD scores in UCSC Genome Browser">Genome Browser</a></li>
                <li class="popover-item"><a class="popover-link" href="/links"
                    title="Links to CADD related resources">Links</a></li>
                <li class="popover-item"><a class="popover-link" href="/contact" title="Contact">Contact</a></li>
              </ul>
            </div>
          </li>
          <li class="navigation-item" height=40>(Alt. site:
            
            <a href="https://cadd.bihealth.org/"><img src="/static/DE-flag.png" width=24
                height=24 style="vertical-align:middle;margin:12px 1px"></a>)
            
          </li>
        </ul>
      </section>
    </nav>

    <section class="container">
      <div id="content">

        <!--<p>
  <font color="#cc9900">
    <strong>A pre-release of CADD v1.7 for genome build GRCh37 - CADD v1.7-pre - is online.</strong><br />
    We provide <strong>genome-wide SNV scores for CADD v1.7</strong>. However, when having multiple VEP
    consequences per variant due to overlapping genes, in this pre-release version we randomly select one of the genes. 
    We recognized that <strong>this behavior can result in potentially lower CADD-scores</strong>, as the more
    severe consequence from the other gene is missed in scoring. We currently recalculate genome-wide scores, providing 
    the most severe transcript consequence for each gene annotated, while provide the highest possible CADD score for all
    annotations lines of such variants (as in previous versions). We expect these files to be ready by the end of 2023. 
    <strong>The CADD v1.7-pre release will be completely deleted afterwards!</strong>. Please use these currently 
    released scores only for exploratory purposes and not in any clinical applications. 
  </font>
</p>-->

<p><font color="darkblue"><strong><i>Note:</i> Scoring of VCF files with CADD v1.7 is still rather slow if many new variants need to be calculated from scratch (e.g., if many insertion/deletion or multinucleotide subsitutions are included). If possible use the pre-scored whole genome and pre-calculated indel files directly where possible. We are very sorry for the inconvenience.</strong></font></p>

<!--<p><font color="darkblue"><strong><i>Note:</i> We are expecting small delays and possibly incomplete files around President's Day (Feb 15, 2016) due to a network file system move. We will keep the webserver running anyways and check for incomplete and missing files after the move is complete on Tuesday (Feb 16). We will automatically re-initiate scoring of those sets. If you believe your download has been incomplete, just download again on Tuesday afternoon (PST).</strong></font></p>-->

<!--
<p><font color="red"><strong>The US site of the CADD webserver and variant scoring will be offline on Jan 8 2024 due to some network maintenance work. Please consider using our alternate site in Germany. Sorry for the inconvenience.</strong></font></p>
-->

<!--<p><font color="darkblue"><strong>The CADD webserver currently works through a long queue of submitted variants. About 36 million variants in more than 380 submission were uploaded since Dec 9 and we are providing limited computational resources to the processing (at most 25 submissions are processed in parallel). Your results will get available eventually and will be available for six weeks after submission. If you are interested in many variants, please consider downloading the pre-calculated files rather than submitting many large requests to the webserver. If you have questions about retrieving large sets of variants without using the webform, please contact us. </strong></font></p>-->

<!-- <p><font color="#ff9900"><strong>IT is working on some network file system issues. It seems that some submissions were not processed because of that (empty output file). We restarted jobs that were clearly incomplete and will monitor processing more closely over the next days. If you encounter unexpected behavior, please send an email to mkircher [at] uw.edu with the submission identifier (name of the link provided after upload/filename of the download).</strong></font></p> -->

<!-- <p><font color="#ff9900"><strong>We are currently updating the CADD webserver. If you encounter unexpected behavior, please be patient and try again. If your problems persist, please send an email to mkircher [at] uw.edu</strong></font></p> -->

<!-- <p><font color="#ff9900"><strong>The CADD webserver has moved to new hardware. Due to some issues with getting everything going on the new hardware, we are watching the server closely right now. If you encounter unexpected behavior, please send an email to mkircher [at] uw.edu</strong></font></p> -->

<!-- <p><font color="red"><strong>The scoring form of the CADD webserver is currently offline as required file shares are currently offline. You can submit your data but it will not be processed until all systems are back online. We were able to establish a workaround for CADD v1.3. Scoring for older releases will only be possible once all systems are back online. Please come again soon. Please also consider using the single variant lookup as well as our files available in the download section. Sorry for the inconvenience. </strong></font></p> -->

<!-- <p><font color="darkblue"><strong>The CADD webserver has a long queue of submitted variants. More than 10 million variants in more than 400 submission were uploaded over the last hours and we are providing limited computational resources to the processing (at most 25 submissions are processed in parallel). Your results will get available eventually and will be available for six weeks after submission. If you are interested in many variants, please consider downloading the pre-calculated files rather than submitting many large requests to the webserver. If you have questions about retrieving large sets of variants without using the webform, please contact us. We would like to find an alternative solution with you.</strong></font> -->

<!--
<p><font color="red"><strong>The CADD webserver is currently offline. You can submit your data but it will not be processed until a filesystem move finishes around 10:00AM PST on July 23 2014. Please come again soon. Sorry for the inconvenience. </strong></font></p>

<p><font color="red">If you uploaded a variant set to our webserver from July 9 2014 13:00 PST to July 10 2014 14:00 PST, your result files were probably empty or at least incomplete. The problem has been fixed. Please download your result file again. You can simply upload the original query file if you no longer have the download link. Sorry for the inconvenience.</font></p>

<p><font color="red">If you uploaded a variant set that included InDels to our webserver from May 11 2014 23:00 PST to May 12 2014 9:00 PST, your result files did not contain the complete set of InDels. The problem has been fixed. Please download your result file again. You can simply upload the original query file if you no longer have the download link. Sorry for the inconvenience.</font></p>

<p><font color="red">Due to a file system outage in the Department of Genome Sciences, it is currently not possible to retrieve variants using our webserver. Please come back soon. If you retrieved variant sets that included InDels in the last hours ( May 28 2014 1:00 AM PST to May 28 2014 10:00 AM PST ), your result files might be incomplete or incorrect. Please score them again once the servers are up again. Sorry for the inconvenience. </font></p>

<p><font color="darkblue"><strong>The CADD webserver is back online in an intermediate set-up. Due to a file system outage in the Department of Genome Sciences our system was offline for a little more than a day. If you retrieved variant sets that included InDels between May 28 2014 1:00 AM PST to May 28 2014 10:00 AM PST, your result files might be incomplete or incorrect. Please download them again. Sorry for the inconvenience. </strong></font></p> -->

        

        <h1>Single nucleotide variant (SNV) lookup</h1>

<p>
  This form allows you to quickly access the score (and annotation) of a single nucleotide variant (SNV) or all scores
  at a specific genomic position.
  If you are investigating multiple or even ranges of CADD SNV scores, please have a look at our <a
    href="/snv-range">Multi-SNV scoring form</a>.
  Please note that copying and downloading of the annotation tables is not working in Internet Explorer.
</p>
<p>
  CADD scores are freely available for all non-commercial applications. If you are
  planning on using them in a commercial application, please <a href="/contact">contact us</a>.
</p>

<p>
<form action="/snv" enctype="multipart/form-data" method="post" name="snv_form">
  <table>
    <tr>
      <th width=25%>Chromosome: </th>
      <th width=25%> <input type="text" name="chr" value=22 size=5 required autofocus></th>
      <th width=25%>Position: </th>
      <th width=25%> <input type="text" name="pos" value=43451447 size=9 required pattern=[0-9]*></th>
    </tr>
    <tr>
      <th>Ref (optional): </th>
      <th> <input type="text" name="ref" placeholder="T" size=5 maxlength=1 pattern="[ACGTacgt]" title="A, T, C or G">
      </th>
      <th>Alt (optional): </th>
      <th> <input type="text" name="alt" placeholder="A" size=5 maxlength=1 pattern="[ACGTacgt]" title="A, T, C or G">
      </th>
    </tr>
    <tr>
      <th>CADD model: </th>
      <th>
        <select name="version" size="1">
          <option value="GRCh38-v1.7" selected>GRCh38-v1.7</option>
          <option value="GRCh37-v1.7">GRCh37-v1.7</option> 
          <option value="GRCh38-v1.6">GRCh38-v1.6</option>
          <option value="GRCh37-v1.6">GRCh37-v1.6</option>
          <option value="GRCh38-v1.5">GRCh38-v1.5</option>
          <option value="GRCh37-v1.4">GRCh37-v1.4</option>
          <option value="GRCh38-v1.4">GRCh38-v1.4</option>
          <option value="v1.3">v1.3</option>
          <option value="v1.2">v1.2</option>
          <option value="v1.1">v1.1</option>
          <option value="v1.0">v1.0</option>
        </select>
      </th>
      <th>
        <label class="container" title='Include underlying annotations in output (not only the score)'>
          <input id='inclAnno' type='checkbox' value='Yes' name='inclAnno'>
          <span class="checkmark checkmark-wide">Include Annotations</span>
        </label>
      </th>
      <th>
        <label class="container" title='Switch rows and columns'>
          <input id='transpose' type='checkbox' value='Yes' name='transpose'>
          <span class="checkmark checkmark-wide">Transpose table</span>
        </label>
      </th>
    </tr>
  </table>
  <p>
    <input type="submit" value="Lookup variant(s)">
  </p>

</form>
</p>



<script>
  formObject = document.forms["snv_form"];
  formObject.elements["chr"].value = "22";
  formObject.elements["pos"].value = "43451447";
  formObject.elements["ref"].value = "T";
  formObject.elements["alt"].value = "A";
  formObject.elements["version"].value = "GRCh38-v1.7";
  if ("None" == "Yes")
    formObject.elements["transpose"].checked = true;
  if ("None" == "Yes")
    formObject.elements["inclAnno"].checked = true;
</script>








<p>
  <font color="#123eab"><strong>The CADD score for your variant is 16.00. (Request: Chromosome 22, Position 43451447, CADD GRCh38-v1.7)</strong></font>
</p>







<p>
<h3>Details:
  <a class="button" id="save-table" onclick="saveTable()"
    title="Export content of SNV table to tsv file. Does not work in IE." style="float: right; margin-left:1rem;">Save
    table</a>
  <button onclick="copySNVtable()" title="Copy content of SNV table to clipboard. Does not work in IE and Edge."
    style="float: right;">Copy table</button>
</h3>
</p>
<p style="clear:both">
<div style="overflow-x:auto">
  <table border=1 id="snv-data" style="width:auto;">
<tr><th><a title="Chromosome">Chrom</a></th><td>22</td></tr>
<tr><th><a title="Position (1-based)">Pos</a></th><td>43451447</td></tr>
<tr><th><a title="Reference allele">Ref</a></th><td>T</td></tr>
<tr><th><a title="Observed allele">Alt</a></th><td>A</td></tr>
<tr><th><a title="Raw score from the model">RawScore</a></th><td>1.842189</td></tr>
<tr><th><a title="CADD PHRED Score">PHRED</a></th><td>16.00</td></tr>
</table>

</div>
</p>
<p>
<h3>Links:</h3>
<ul>
  <li><a href="/snv/GRCh38-v1.7/22:43451447_T_A">Direct link</a></li>
<li><a target="_blank" href="https://bravo.sph.umich.edu/freeze5/hg38/variant/22-43451447-T-A">Bravo variant</a></li>
<li><a target="_blank" href="https://gnomad.broadinstitute.org/variant/22-43451447-T-A?dataset=gnomad_r4">gnomAD variant</a></li>
<li><a target="_blank" href="https://bravo.sph.umich.edu/freeze5/hg38/region/22-43451447-43451447">Bravo browser</a></li>
<li><a target="_blank" href="https://gnomad.broadinstitute.org/region/22-43451447-43451447?dataset=gnomad_r4">gnomAD browser</a></li>
<li><a target="_blank" href="https://genome.ucsc.edu/cgi-bin/hgTracks?db=hg38&highlight=hg38.chr22:43451447-43451447&position=chr22:43451422-43451472">UCSC Genome Browser</a></li>
<li><a target="_blank" href="https://www.ensembl.org/Homo_sapiens/Location/View?r=22:43451447-43451447">Ensembl genome browser</a></li>
<li><a target="_blank" href="https://www.ncbi.nlm.nih.gov/genome/gdv/browser/?context=genome&id=GCF_000001405.38&chr=22&from=43451422&to=43451472&mk=chr22:43451447-43451447|SNV">NCBI Genome Data Viewer</a></li>
</ul>
</p>

<script src="/static/jquery-3.3.1.min.js"></script>
<script>
  function copySNVtable() {
    var el = document.getElementById('snv-data');
    var body = document.body, range, sel;
    if (document.createRange && window.getSelection) {
      range = document.createRange();
      sel = window.getSelection();
      sel.removeAllRanges();
      try {
        range.selectNodeContents(el);
        sel.addRange(range);
      } catch (e) {
        range.selectNode(el);
        sel.addRange(range);
      }
    } else if (body.createTextRange) {
      range = body.createTextRange();
      range.moveToElementText(el);
      range.select();
    }
    document.execCommand("copy");
  };
  function saveTable() {
    var retContent = [];
    var retString = '';
    var el = document.getElementById('snv-data');
    $(el).find('tr').each(function (idx, elem) {
      var elemText = [];
      $(elem).children('*').each(function (childIdx, childElem) {
        elemText.push($(childElem).text());
      });
      retContent.push(`${elemText.join('\t')}`);
    });
    retString = retContent.join('\r\n');
    var file = new Blob([retString], { type: 'text/plain' });
    var btn = $('#save-table');
    btn.attr("href", URL.createObjectURL(file));
    btn.prop("download", "CADD-GRCh38-v1.7-22-43451447.tsv");
  }
</script>



        

      </div>
    </section>

    <footer class="footer">
      <section class="container">
        
        <p align="center">&copy; University of Washington, Hudson-Alpha Institute for Biotechnology and Berlin Institute
          of Health at Charité - Universitätsmedizin Berlin 2013-2023. All rights reserved.</p>
        
        <p align="center"><a href="http://www.washington.edu/online/terms/">Terms and Conditions</a> and the <a
            href="http://www.washington.edu/online/privacy/">Online Privacy Statement</a> of the University of
          Washington apply. </p>
        
      </section>
    </footer>

  </main>
</body>
<script>
  "use strict";
  !function () {
    function e(e) {
      for (r = 0;
        r < n.length;
        r++)n[r].classList.remove("popover-open")
    }
    function t(t) {
      t.preventDefault(),
        document.querySelector(this.getAttribute("href")).classList.contains("popover-open") ? document.querySelector(this.getAttribute("href")).classList.remove("popover-open") : (e(), document.querySelector(this.getAttribute("href")).classList.add("popover-open")), t.stopImmediatePropagation()
    }
    var o = document.querySelectorAll("[data-popover]"),
      n = document.querySelectorAll(".popover"),
      r = void 0;
    for (r = 0;
      r < o.length;
      r++)o[r].addEventListener("click", t);
    document.addEventListener("click", e)
  }()
  
  	  ,
  function(i, s, o, g, r, a, m) {
    i['GoogleAnalyticsObject'] = r; i[r] = i[r] || function () {
      (i[r].q = i[r].q || []).push(arguments)
    }, i[r].l = 1 * new Date(); a = s.createElement(o),
      m = s.getElementsByTagName(o)[0]; a.async = 1; a.src = g; m.parentNode.insertBefore(a, m)
  } (window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
  ga('create', 'UA-4137655-2', 'washington.edu');
  ga('send', 'pageview');
  
</script>

</html>